src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -376,8 +377,11 @@ class ManualColumnResize extends BasePlugin {
    * @param {MouseEvent} event The mouse event.
    */
   onMouseOver(event) {
-    if (this.checkIfColumnHeader(event.target)) {
-      const th = this.getTHFromTargetElement(event.target);
+    // Workaround for #6926 - if the `event.target` is temporarily detached, we can go straight the the parent.
+    if (this.checkIfColumnHeader(isDetached(event.target) ? getTargetParent(event) : event.target)) {
+      // Because of a problem described in #6926, the `event.target` element is sometimes (temporarily) detached.
+      // This workaround prevents the error from #6926 happening.
+      const th = event.target.tagName === 'TH' ? event.target : this.getClosestTHParent(getTargetParent(event));
```

Comment:
```suggestion
    if (isDetached(event.target)) {
      return;
    }

    if (this.checkIfColumnHeader(event.target)) {
      const th = this.getClosestTHParent(event.target);
```
I don't see any advantages of using the `composedPath` (in the form of `getTargetParent` helper) method of the event object. IMO skipping the detached elements should be enough for this fix. I've compared how the two approaches behave and didn't see any differences. Moreover, the usage of the mentioned API would increase code complexity.

In both cases, the error disappears. Sometimes in both cases, the handler is not showing at all.

### Playing with PR changes:
for rows:
![PR](https://user-images.githubusercontent.com/571316/96242868-40bb4500-0fa4-11eb-9eb9-5f7c0b8de96a.gif)
for columns:
![PR-columns](https://user-images.githubusercontent.com/571316/96242878-4749bc80-0fa4-11eb-9fad-6566657e6807.gif)

### Playing with applied skipped detached nodes:
for rows:
![Local](https://user-images.githubusercontent.com/571316/96243020-74966a80-0fa4-11eb-94f3-1a9ac4f9074d.gif)
for columns:
![Local-columns](https://user-images.githubusercontent.com/571316/96243044-7b24e200-0fa4-11eb-8809-b0b020080406.gif)

Reply:
You might be right - this problem happens on `mouseover`, so after skipping one event on a detached element, the next one will probably be triggered on an attached element. I'll do some additional testing, but your suggestion looks promising 👍